zygrader/main.py

```diff
@@ -108,7 +108,8 @@ def view_changelog():
     lines = config.versioning.load_changelog()
 
     popup = ui.layers.ListLayer("Changelog", popup=True)
-    popup.set_exit_text("Close")
+    popup.set_exit_text("Press enter to close")
```

Comment:
Technically, escape works to close too. I really don't think we need to say that here, but if you can think of a concise way to include it, we could.
I do like this better than "Press enter or escape to close" just because adding "escape" makes it way too wordy.

Reply:
Oh, and also left. But again, it would probably be more verbose than helpful unless we can find a concise way to say "Press enter or escape or left (if you enable left/right arrow navigation) to close"

Reply:
I think only mentioning Enter here is fine. Those who enable `ESC` for going back know it exists, and left arrow becomes natural for those who use it.